packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1049,9 +1052,9 @@ export function useGlobalStylesOutput() {
 			hasFallbackGapSupport,
 			disableLayoutStyles
 		);
+		const svgs = toSvgFilters( mergedConfig, blockSelectors );
```

Comment:
The definition of this function changed from returning React elements to returning a string. The function was exported from this file, but it isn't exported from the package, so I think the API change here is fine.